src/App.tsx

```diff
@@ -10,33 +12,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
+  const selectedTabId = (tab: Tab): void => {
+    setSelectedTab(tab);
+  };
```

Comment:
```suggestion
  const handleTabSelected = (tab: Tab): void => {
    setSelectedTab(tab);
  };
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364